.changeset/chilly-pumas-mate.md

```diff
@@ -0,0 +1,10 @@
+---
+"@lynx-js/react": patch
+---
+
+Remove the `@lynx-js/react/testing-library/vitest-config` exports, user should replace it with the new `vitestTestingLibraryPlugin` in `vitest.config.js`:
```

Comment:
This is a **BREAKING CHANGE**, we should use `minor` change and mark **BREAKING CHANGE** in changeset.

BTW, can we avoid making breaking change (i.e.: keep the `createVitestConfig`)